Bottom: 0c85af868f0bded0ac09890ebd9a7cd8a5361bfc
Top:    aa80ae9351a25d3177adf889cf45912b140a8b25
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-29 10:41:31 +0800

Refresh of tuntap-rename-struct-0

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index f7d6708..0853829 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -915,7 +915,6 @@ static int run_filter(struct tap_filter *filter, const struct sk_buff *skb)
 	return 0;
 }
 
-
 /*
  * Checks whether the packet is accepted or not.
  * Returns: 0 - drop, !=0 - accept
